app/src/main/java/org/mozilla/fenix/customtabs/FennecWebAppIntentProcessor.kt

```diff
@@ -113,6 +116,10 @@ class FennecWebAppIntentProcessor(
 
             WebAppManifestParser().parse(manifestField).getOrNull()
         } catch (e: IOException) {
+            logger.error("Failed to parse web app manifest due to IOException", e)
+            null
+        } catch (e: JSONException) {
+            logger.error("Failed to parse web app manifest due to JSONException", e)
```

Comment:
We're still missing multi-catch in Kotlin! ☕

Reply:
Yeah :)